libs/designer/src/lib/core/actions/bjsworkflow/serializer.ts

```diff
@@ -58,26 +58,29 @@ export interface SerializeOptions {
 }
 
 export const serializeWorkflow = async (rootState: RootState, options?: SerializeOptions): Promise<Workflow> => {
```

Comment:
@Eric-B-Wu There are two places in LAUX repo where skipValidation is passed as "true". Can you please also confirm this wouldn't break those scenarios? Or I can update them to not pass the value for now since it wasn't being used before anyway.

https://github.com/Azure/LogicAppsUX/blob/b14642c7187a3b06907c270df9a60a3d364d534b/apps/designer-standalone/src/app/AzureLogicAppsDesigner/DesignerCommandBar.tsx#:~:text=skipValidation%3A%20true%2C

https://github.com/Azure/LogicAppsUX/blob/a8a9471148437222de7eb84af89ebc9479346f3c/apps/vs-code-designer-react/src/app/DesignerCommandBar/index.tsx#:~:text=skipValidation%3A%20true%2C

Reply:
Updated to "skipValidation: false" in vs-code-designer-react as per discussion with Eric